tianshou/data/batch.py

```diff
@@ -554,7 +565,7 @@ def stack_(self,
                         self.__dict__[k][i] = val
 
     @staticmethod
-    def stack(batches: List['Batch'], axis: int = 0) -> 'Batch':
+    def stack(batches: List[Union[dict, 'Batch']], axis: int = 0) -> 'Batch':
```

Comment:
Why to support `dict` ? Just in case ? Not supporting it would avoid having to check key type. But as you wish.

Reply:
To keep in consistent with ``cat``.